.gitlab-ci.yml

```diff
@@ -16,26 +18,26 @@ variables:
   # some useful values
   COMPILER_32BIT: "4.02.3+32bit"
 
-  COMPILER_BLEEDING_EDGE: "4.06.0"
-  CAMLP5_VER_BLEEDING_EDGE: "7.03"
+  COMPILER_BLEEDING_EDGE: "4.06.1"
+  CAMLP5_VER_BLEEDING_EDGE: "7.05"
 
-  TIMING_PACKAGES: "time python"
+  TIMING_PACKAGES: "time python3"
 
   COQIDE_PACKAGES: "libgtk2.0-dev libgtksourceview2.0-dev"
   #COQIDE_PACKAGES_32BIT: "libgtk2.0-dev:i386 libgtksourceview2.0-dev:i386"
-  COQIDE_OPAM: "lablgtk-extras"
-  COQIDE_OPAM_BE: "lablgtk.2.18.6 lablgtk-extras.1.6"
-  COQDOC_PACKAGES: "texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-math-extra texlive-fonts-recommended texlive-fonts-extra latex-xcolor ghostscript transfig imagemagick tipa python3-pip"
-  COQDOC_OPAM: "hevea"
-  SPHINX_PACKAGES: "bs4 sphinx sphinx_rtd_theme pexpect antlr4-python3-runtime sphinxcontrib-bibtex"
+  SPHINX_PACKAGES: "antlr4-python3-runtime"
+  COQIDE_OPAM: "lablgtk.2.18.5 conf-gtksourceview.2"
+  COQIDE_OPAM_BE: "lablgtk.2.18.6 conf-gtksourceview.2"
+  COQDOC_PACKAGES: "texlive-latex-recommended texlive-fonts-recommended transfig imagemagick hevea python3-sphinx python3-pexpect python3-sphinx-rtd-theme python3-bs4 python3-sphinxcontrib.bibtex python3-pip"
   ELPI_OPAM: "elpi"
 
-
 before_script:
+  - cat /proc/{cpu,mem}info || true
```

Comment:
??

Reply:
> ??

I guess this is clear; I want to get more information about runners; there are quite wild differences; suggestions welcome, including using `docker` client commands.